pkg/config/config.go

```diff
@@ -0,0 +1,109 @@
+package config
+
+import (
+	"strconv"
+
+	"github.com/pelletier/go-toml"
+)
+
+// DefaultConfig for the DC/OS CLI.
```

Comment:
What is the purpose of this `var`? Why a Timeout of `180`? The code already describes what is currently in the comment.